src/dotnet/Core/IO/CancellableDebouncer.cs

```diff
@@ -0,0 +1,126 @@
+namespace ActualChat.IO;
+
+public class CancellableDebouncer<T> : WorkerBase
```

Comment:
It's not obvious when to use CancellableDebouncer<T> and when Debouncer.

Reply:
added Debouncer.New...